bin/generate-atomic-docs.rb

```diff
@@ -135,11 +135,11 @@ def generate_attack_matrix!(title_prefix, output_doc_path, only_platform: /.*/)
   #
   # Generates a master Markdown index of ATT&CK Tactic -> Technique -> Atomic Tests
   #
-  def generate_index!(title_prefix, output_doc_path, only_platform: /.*/)
+  def generate_index!(title_prefix, output_doc_path, only_platform: /.*/, attack_platform: /.*/)
     result = ''
     result += "# #{title_prefix} Atomic Tests by ATT&CK Tactic & Technique\n"
 
-    ATTACK_API.techniques_by_tactic(only_platform: only_platform).each do |tactic, techniques|
+    ATTACK_API.techniques_by_tactic(only_platform: attack_platform).each do |tactic, techniques|
```

Comment:
MITRE renamed Azure AD to Identity Provider and Office 365, Google Workspace to Office Suite. I had to use different filters for ATTACK_API and ATOMIC_RED_TEAM ruby classes.